operators/pkg/tenant-controller/tenant_controller.go

```diff
@@ -307,6 +311,86 @@ func (r *TenantReconciler) checkValidWorkspaces(ctx context.Context, tn *crownla
 	return tenantExistingWorkspaces, workspaces, err
 }
 
+// deleteClusterNamespace deletes the namespace for the tenant, if it fails then it returns an error.
+func (r *TenantReconciler) deleteClusterNamespace(ctx context.Context, tn *crownlabsv1alpha2.Tenant, nsName string) (err error) {
```

Comment:
```suggestion
func (r *TenantReconciler) deleteClusterNamespace(ctx context.Context, tn *crownlabsv1alpha2.Tenant, nsName string) error {
```

Reply:
Resolved in [c8c8301](https://github.com/netgroup-polito/CrownLabs/pull/821/commits/c8c83013ca6fbec5de81eeb3067f9002ded4e23a)